FantasmoSDK/Classes/Network/FMApi.swift

```diff
@@ -165,9 +167,20 @@ class FMApi {
             parameters: params,
             token: token,
             completion: postCompletion,
-            error: postError)
+            error: postError
+        )
+    }
+    
+    func sendDeviceCharacteristicsRequest() {
```

Comment:
As I have explained on Slack, we don't know if this will be another request or merely a field in an existing request. Please revert the FMApi changes. They should not be part of this ticket or PR.

Reply:
Ok, can I leave renaming `localize()` -> `sendLocalizationRequest()`